Add OLS-9 blog (#811)
open-life-science/open-life-science.github.io: _posts/2024-07-15-openseeds-in-2024.md

---
layout: post
title: 'Open Seeds in 2024: one cohort, two streams: OLS-9, and CAT-1'
authors:
- yochannah
image: https://images.unsplash.com/photo-1486868018400-960892a825f4
photos:
  name: Nathan Anderson
  license: CC BY-SA 4.0
  url: https://images.unsplash.com/photo-1486868018400-960892a825f4
---

## How do we sustain open science and open research as a business? 

> 2024 has been a year of change for OLS. Our original CZI core funding is winding up, with all four of the cohorts (OLS-5 to OLS-8) now completed. 

Reviewer: ```suggestion

2024 has been a year of change for OLS. Our original CZI core funding is winding up, with all four of the cohorts (OLS-5 to OLS-8) now completed. 
```

## More diverse income streams, and a two-part OLS-9

The next cohort of Open Seeds will have a mixed model for delivery: partly grant-based, and partly contract-based. Tajuddeen will coordinate the Open Seeds cohort for [Catalyst project members]({% link catalyst/index.md %}). Applications are open to Catalyst project members in Latin America and Africa. 

In addition, we're quite proud to share that the [Digital Research Alliance Canada](https://alliancecan.ca/en/latest/news/dri-edia-champions-new-funding-opportunity-advance-edia-dri) has contracted OLS to offer a full cohort to EDIA-focussed grantees based in Canada - **the call for applications for $35,000 CAD and participation in the Open Seeds program is open until 17 July**. 

Malvika Sharan will co-ordinate the DRA-Canada stream of work, alongside the [OLS-9 Resident Fellow](LINK TO THE RECRUITMENT BLOG) - name coming soon. 

## When is the next open-application cohort coming up? 

> We're still on the hunt for funders to support OLS-10 and onwards, whether service-based contract or grant-based. Interested? A full in-house 16-week cohort for a high-income-country based organisation is usually around $200,000 USD. This supports up to 45 mentors, 30 speakers, and 45 teams/individual mentees. It also builds in the participation costs for microgrants for participants in lower/middle-income settings. Contact (team, Yo or Malvika) at [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }}) to discuss options. 

Reviewer: ```suggestion
We're still on the hunt for funders to support OLS-10 and onwards, whether service-based contract or grant-based. 

Interested? A **full in-house 16-week cohort** for a high-income-country based organisation is usually around $200,000 USD. This supports up to **45 mentors, 30 speakers, and 45 teams/individual mentees**. It also builds in the participation costs for **microgrants for participants in lower/middle-income** settings. Contact (team, Yo or Malvika) at [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }}) to discuss options. 
```

---
layout: post
title: 'Open Seeds in 2024: one cohort, two streams: OLS-9, and CAT-1'
authors:
- yochannah
image: https://images.unsplash.com/photo-1486868018400-960892a825f4
photos:
  name: Nathan Anderson
  license: CC BY-SA 4.0
  url: https://images.unsplash.com/photo-1486868018400-960892a825f4
---

## How do we sustain open science and open research as a business? 

2024 has been a year of change for OLS. Our original CZI core funding is winding up, with all four of the cohorts (OLS-5 to OLS-8) now completed. 

> We weren't too worried sustainability until at least 2026, through our [NASA grants]({% link _posts/2023-05-05-NASA-grant-announcement.md %}), which now support the [5-week Nebula program]({% link nebula/index.md %}). Once [OLS-8]({% link openseeds/ols-8/index.md%}) wrapped up, though, we weren't entirely sure how to continue running [Open Seeds]({% link openseeds/index.md %}), even though it was our original, core program. 

Author: ```suggestion
We weren't too worried sustainability until at least 2026, through our [NASA grants]({% link _posts/2023-05-05-NASA-grant-announcement.md %}), which now support the [5-week Nebula program]({% link nebula/index.md %}). Once [OLS-8]({% link openseeds/ols-8/index.md %}) wrapped up, though, we weren't entirely sure how to continue running [Open Seeds]({% link openseeds/index.md %}), even though it was our original, core program. 
```

Author: @yochannah - Please read the first sentence of this paragraph. Think we're missing an "about" after **worried**?

Author: Resolved by Malvika's ingenuity.

## More diverse income streams, and a two-part OLS-9

The next cohort of Open Seeds will have a mixed model for delivery: partly grant-based, and partly contract-based. 

> Tajuddeen will coordinate the Open Seeds cohort for [Catalyst project members]({% link catalyst/index.md %}). Applications are open to Catalyst project members in Latin America and Africa. 

Author: ```suggestion
Tajuddeen will coordinate the Open Seeds cohort for [Catalyst project members]({% link catalyst/cat-1/schedule.md %}). Applications are open to Catalyst project members in Latin America and Africa. 
```

authors:
- yochannah
image: https://images.unsplash.com/photo-1486868018400-960892a825f4
photos:
  name: Nathan Anderson
  license: CC BY-SA 4.0
  url: https://images.unsplash.com/photo-1486868018400-960892a825f4
---

# OLS-9 Launches with Two Tracks in Partnership with Digital Research Alliance of Canada and Catalyst Project

We are delighted to announce that we have partnered with the Digital Research Alliance of Canada (the Alliance) and the Catalyst Project to deliver two tracks as part of OLS's Open Seeds program, OLS-9.

## OLS-9 DRI-EDIA Track

We are proud to share that the [Digital Research Alliance Canada](https://alliancecan.ca/en/funding-opportunities/dri-edia-champions-pilot-program) has contracted OLS to offer a full cohort to EDIA-focused grantees based in Canada.
The DRI EDIA Champions Pilot Program aims to increase awareness and uptake of the Alliance’s digital research infrastructure (DRI) by equity, diversity, inclusion, and accessibility (EDIA) champions across Canadian institutions.

Project leads/mentees in the "**OLS-9 DRI EDIA Track**" will be fellows selected under the DRI EDIA Champions Program.

> Malvika Sharan will lead the delivery of this track. She will be joined by a Resident Fellow, [Seun Olufemi]({% link people.html# %}). 

Author: ```suggestion
Malvika Sharan will lead the delivery of this track. She will be joined by a Resident Fellow, [Seun Olufemi]({% link people.html %}). 
```
